Neos.Neos/Classes/Domain/Service/TypoScriptService.php

```diff
@@ -45,31 +45,39 @@ class TypoScriptService
      *
      * @var string
      */
-    protected $siteRootTypoScriptPattern = 'resource://%s/Private/TypoScript/Root.ts2';
+    protected $siteRootTypoScriptPattern = 'resource://%s/Private/TypoScript/Root.fusion';
 
     /**
-     * Pattern used for determining the TypoScript root file for a site
+     * Legacy pattern used for determining the TypoScript root file for a site
      *
      * @var string
-     * @deprecated since 1.2 will be removed in 2.0
+     * @deprecated since 3.0 will be removed in 4.0
      */
-    protected $legacySiteRootTypoScriptPattern = 'resource://%s/Private/TypoScripts/Library/Root.ts2';
+    protected $legacySiteRootTypoScriptPattern = 'resource://%s/Private/TypoScript/Root.ts2';
 
     /**
      * Pattern used for determining the TypoScript root file for autoIncludes
      *
      * @var string
      */
-    protected $autoIncludeTypoScriptPattern = 'resource://%s/Private/TypoScript/Root.ts2';
+    protected $autoIncludeTypoScriptPattern = 'resource://%s/Private/TypoScript/Root.fusion';
```

Comment:
So this PR should just provide the support for .fusion files and then go on with the next tasks